source/disassemble.cpp

```diff
@@ -134,7 +139,6 @@ spv_result_t Disassembler::HandleHeader(spv_endianness_t endian,
   endian_ = endian;
 
   if (header_) {
-    SetGrey();
```

Comment:
This has always come out as black for me, so it just looks like 5 blank lines at the top of the entry. Removing this just leaves it as the default colour which seems to work better.

Reply:
Right. The bad behaviour is on macOS.  I've just tolerated it. But it's bad and we should remove this.